mbtiles/src/pool.rs

```diff
@@ -26,9 +37,303 @@ impl MbtilesPool {
+          minzoom: 0
+          name: Major cities from Natural Earth data
+          version: "2"
+          format: pbf
+        json:
+          tilestats:
+            layerCount: 1
+            layers:
+              - attributeCount: 1
+                attributes:
+                  - attribute: name
+                    count: 68
+                    type: string
+                    values:
+                      - Addis Ababa
+                      - Amsterdam
+                      - Athens
+                      - Atlanta
+                      - Auckland
+                      - Baghdad
+                      - Bangalore
+                      - Bangkok
+                      - Beijing
+                      - Berlin
+                      - Bogota
+                      - Buenos Aires
+                      - Cairo
+                      - Cape Town
+                      - Caracas
+                      - Casablanca
+                      - Chengdu
+                      - Chicago
+                      - Dakar
+                      - Denver
+                      - Dubai
+                      - Geneva
+                      - Hong Kong
+                      - Houston
+                      - Istanbul
+                      - Jakarta
+                      - Johannesburg
+                      - Kabul
+                      - Kiev
+                      - Kinshasa
+                      - Kolkata
+                      - Lagos
+                      - Lima
+                      - London
+                      - Los Angeles
+                      - Madrid
+                      - Manila
+                      - Melbourne
+                      - Mexico City
+                      - Miami
+                      - Monterrey
+                      - Moscow
+                      - Mumbai
+                      - Nairobi
+                      - New Delhi
+                      - New York
+                      - Paris
+                      - Rio de Janeiro
+                      - Riyadh
+                      - Rome
+                      - San Francisco
+                      - Santiago
+                      - Seoul
+                      - Shanghai
+                      - Singapore
+                      - Stockholm
+                      - Sydney
+                      - São Paulo
+                      - Taipei
+                      - Tashkent
+                      - Tehran
+                      - Tokyo
+                      - Toronto
+                      - Vancouver
+                      - Vienna
+                      - "Washington, D.C."
+                      - Ürümqi
+                      - Ōsaka
+                count: 68
+                geometry: Point
+                layer: cities
+        agg_tiles_hash: D4E1030D57751A0B45A28A71267E46B8
+        "#);
+        assert_eq!(pool.detect_type().await.unwrap(), MbtType::FlatWithHash);
+        let t1 = pool.get_tile(0, 0, 0).await.unwrap().unwrap();
```

Comment:
no clue why all of the sudden getting a tile for this fails.
Will look at this tomorrow/in the evening.

Weird

Reply:
Reason was that 0,0,0 never existed in said mbtiles archive.